Update 4 (#251)
NilPointer-Software/bdfd-wiki: src/bdscript/author.md

# $author
Adds author text to an embed.

## Syntax
```
$author[text;(index)]
```

## Parameters
- `text`: The text that appears in the author text. It cannot exceed more than 256 characters.

> - `index`: What embed the author text should belong to. _(Optional)_ [(learn more)](../resources/embedIndexes.md)

Reviewer: ```suggestion
- `index`: To which embed the author text will be added. _(Optional)_ [(learn more)](../resources/embedIndexes.md)
```
